ServiceRequest/src/main/java/com/polus/servicerequest/entity/User.java

```diff
@@ -22,38 +22,38 @@ public class User {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Column(name = "userid")
+	@Column(name = "USERID")
```

Comment:
Use separation for naming conversation. Eg : firstName means FIRST_NAME

Reply:
Do i need to change the variables in the entity or only the entity